{
  "version": "4.22.3",
  "time": 1.328,
  "errors": [],
  "tasks": [
    {
      "resource": {
        "name": "ft_convenio_entrada_dotacao",
        "title": "Convênio Entrada Dotação",
        "path": "data/ft_convenio_entrada_dotacao.csv.gz",
        "schema": {
          "fields": [
            {
              "name": "id_tempo",
              "title": "Tempo - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_convenio",
              "title": "Convênio de Entrada - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_orgao",
              "title": "Órgão - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_concedente",
              "title": "Concedente - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_programa",
              "title": "Programa - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_acao",
              "title": "Ação Orçamentária - Chave",
              "type": "string",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_elemento",
              "title": "Elemento de Despesa - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_unidade_orc",
              "title": "Unidade Orçamentária - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "dotacao_orcamentaria",
              "title": "Dotação Orçamentária",
              "type": "string",
              "format": "default",
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "ano_particao",
              "title": "Ano",
              "type": "year",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_concedente_atual",
              "title": "Valor Concedente Atualiado",
              "description": "Valor financeiro previsto no convênio para o concedente repassar ao proponente, incluindo termos aditivos.\n",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ",",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_proponente_atual",
              "title": "Valor Proponente Atualizado",
              "description": "Valor financeiro previsto no convênio para o proponente dar como contrapartida financeira, incluindo termos aditivos.\n",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ",",
              "constraints": {
                "required": true
              }
            }
          ],
          "missingValues": [
            "SEM DESCRICAO"
          ],
          "foreignKeys": [
            {
              "fields": [
                "id_tempo"
              ],
              "reference": {
                "fields": [
                  "id_tempo"
                ],
                "resource": "dm_tempo_diario"
              }
            },
            {
              "fields": [
                "id_concedente"
              ],
              "reference": {
                "fields": [
                  "id_concedente"
                ],
                "resource": "dm_concedente"
              }
            },
            {
              "fields": [
                "id_orgao"
              ],
              "reference": {
                "fields": [
                  "id_orgao"
                ],
                "resource": "dm_orgao_proponente"
              }
            },
            {
              "fields": [
                "id_elemento"
              ],
              "reference": {
                "fields": [
                  "id_elemento"
                ],
                "resource": "dm_elemento_desp"
              }
            },
            {
              "fields": [
                "id_programa"
              ],
              "reference": {
                "fields": [
                  "id_programa"
                ],
                "resource": "dm_programa_conv_ent"
              }
            },
            {
              "fields": [
                "id_convenio"
              ],
              "reference": {
                "fields": [
                  "id_convenio"
                ],
                "resource": "dm_convenio_entrada"
              }
            }
          ]
        },
        "sources": [
          {
            "title": "Tabela origem no schema AGE7 no banco de dados do Portal da Transparência",
            "table": "ft_convenio_entrada_dotacao"
          }
        ],
        "format": "csv",
        "mediatype": "text/csv",
        "encoding": "utf-8-sig",
        "dialect": {
          "delimiter": ";"
        },
        "profile": "tabular-data-resource",
        "scheme": "file",
        "hashing": "md5",
        "compression": "gz",
        "stats": {
          "hash": "ad429b5cebc06bee508073d1c043fe9c",
          "bytes": 81155,
          "fields": 12,
          "rows": 5599
        }
      },
      "time": 1.328,
      "scope": [
        "hash-count-error",
        "byte-count-error",
        "field-count-error",
        "row-count-error",
        "blank-header",
        "extra-label",
        "missing-label",
        "blank-label",
        "duplicate-label",
        "incorrect-label",
        "blank-row",
        "primary-key-error",
        "foreign-key-error",
        "extra-cell",
        "missing-cell",
        "type-error",
        "constraint-error",
        "unique-error"
      ],
      "partial": false,
      "errors": [
        {
          "cells": [
            "50274",
            "1728",
            "57",
            "515",
            "17585",
            "",
            "498",
            "4159",
            "1250 06.181.034.4048.1 3 1 90 12 1.0.0",
            "2021",
            "0.00",
            "3134.64"
          ],
          "rowNumber": 66,
          "rowPosition": 67,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"67\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1728",
            "57",
            "515",
            "17585",
            "",
            "469",
            "4159",
            "1250 06.181.034.4048.1 3 3 90 30 1.70.1",
            "2021",
            "10319.72",
            "0.00"
          ],
          "rowNumber": 67,
          "rowPosition": 68,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"68\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1728",
            "57",
            "515",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.70.1",
            "2021",
            "5353.46",
            "0.00"
          ],
          "rowNumber": 68,
          "rowPosition": 69,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"69\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49920",
            "1725",
            "16",
            "416",
            "17609",
            "",
            "459",
            "4165",
            "1400 06.182.155.4472.1 4 4 90 52 1.0.0",
            "2021",
            "0.00",
            "40449.33"
          ],
          "rowNumber": 122,
          "rowPosition": 123,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"123\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49920",
            "1725",
            "16",
            "416",
            "17609",
            "",
            "459",
            "4165",
            "1400 06.182.155.4472.1 4 4 90 52 1.24.1",
            "2021",
            "250000.00",
            "0.00"
          ],
          "rowNumber": 123,
          "rowPosition": 124,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"124\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49920",
            "1724",
            "16",
            "416",
            "17609",
            "",
            "459",
            "4165",
            "1400 06.182.155.4472.1 4 4 90 52 1.0.0",
            "2021",
            "0.00",
            "265534.00"
          ],
          "rowNumber": 159,
          "rowPosition": 160,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"160\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49920",
            "1724",
            "16",
            "416",
            "17609",
            "",
            "459",
            "4165",
            "1400 06.182.155.4472.1 4 4 90 52 1.24.1",
            "2021",
            "10000000.00",
            "0.00"
          ],
          "rowNumber": 160,
          "rowPosition": 161,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"161\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50283",
            "1732",
            "40",
            "103",
            "17646",
            "",
            "534",
            "4089",
            "1260 12.302.026.1024.1 3 3 90 39 1.70.1",
            "2021",
            "763013.13",
            "0.00"
          ],
          "rowNumber": 1275,
          "rowPosition": 1276,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1276\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1734",
            "57",
            "377",
            "17585",
            "",
            "498",
            "4159",
            "1250 06.181.034.4048.1 3 1 90 12 1.0.0",
            "2021",
            "0.00",
            "4000.00"
          ],
          "rowNumber": 1451,
          "rowPosition": 1452,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1452\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1734",
            "57",
            "377",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.70.1",
            "2021",
            "200000.00",
            "0.00"
          ],
          "rowNumber": 1452,
          "rowPosition": 1453,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1453\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50269",
            "1723",
            "57",
            "15",
            "17585",
            "",
            "498",
            "4159",
            "1250 06.181.034.4057.1 3 1 90 12 1.0.0",
            "2021",
            "0.00",
            "30000.00"
          ],
          "rowNumber": 1490,
          "rowPosition": 1491,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1491\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50269",
            "1723",
            "57",
            "15",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4057.1 4 4 90 52 1.70.1",
            "2021",
            "150000.00",
            "0.00"
          ],
          "rowNumber": 1491,
          "rowPosition": 1492,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1492\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1726",
            "57",
            "158",
            "17585",
            "",
            "498",
            "4159",
            "1250 06.181.034.4048.1 3 1 90 12 1.0.0",
            "2021",
            "0.00",
            "3134.64"
          ],
          "rowNumber": 1551,
          "rowPosition": 1552,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1552\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1726",
            "57",
            "158",
            "17585",
            "",
            "469",
            "4159",
            "1250 06.181.034.4048.1 3 3 90 30 1.70.1",
            "2021",
            "10319.72",
            "0.00"
          ],
          "rowNumber": 1552,
          "rowPosition": 1553,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1553\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1726",
            "57",
            "158",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.70.1",
            "2021",
            "5353.46",
            "0.00"
          ],
          "rowNumber": 1553,
          "rowPosition": 1554,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1554\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1729",
            "57",
            "1234",
            "17585",
            "",
            "498",
            "4159",
            "1250 06.181.034.4048.1 3 1 90 12 1.0.0",
            "2021",
            "0.00",
            "1745.37"
          ],
          "rowNumber": 1582,
          "rowPosition": 1583,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1583\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1729",
            "57",
            "1234",
            "17585",
            "",
            "498",
            "4159",
            "1250 06.181.034.4048.1 3 1 90 12 1.0.0",
            "2021",
            "0.00",
            "1745.37"
          ],
          "rowNumber": 1582,
          "rowPosition": 1583,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1583\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1729",
            "57",
            "1234",
            "17585",
            "",
            "469",
            "4159",
            "1250 06.181.034.4048.1 3 3 90 30 1.70.1",
            "2021",
            "3373.41",
            "0.00"
          ],
          "rowNumber": 1583,
          "rowPosition": 1584,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1584\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1729",
            "57",
            "1234",
            "17585",
            "",
            "469",
            "4159",
            "1250 06.181.034.4048.1 3 3 90 30 1.70.1",
            "2021",
            "3373.41",
            "0.00"
          ],
          "rowNumber": 1583,
          "rowPosition": 1584,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1584\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1729",
            "57",
            "1234",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.70.1",
            "2021",
            "5353.46",
            "0.00"
          ],
          "rowNumber": 1584,
          "rowPosition": 1585,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1585\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50274",
            "1729",
            "57",
            "1234",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.70.1",
            "2021",
            "5353.46",
            "0.00"
          ],
          "rowNumber": 1584,
          "rowPosition": 1585,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1585\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50262",
            "1730",
            "40",
            "103",
            "17646",
            "",
            "469",
            "4089",
            "1260 12.302.026.1024.1 3 3 90 30 1.70.1",
            "2021",
            "500000.00",
            "0.00"
          ],
          "rowNumber": 1585,
          "rowPosition": 1586,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1586\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50262",
            "1730",
            "40",
            "103",
            "17646",
            "",
            "534",
            "4089",
            "1260 12.302.026.1024.1 3 3 90 39 1.70.1",
            "2021",
            "460000.00",
            "0.00"
          ],
          "rowNumber": 1586,
          "rowPosition": 1587,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1587\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49901",
            "1727",
            "57",
            "1209",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.0.0",
            "2021",
            "0.00",
            "47525.58"
          ],
          "rowNumber": 1587,
          "rowPosition": 1588,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1588\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49901",
            "1727",
            "57",
            "1209",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.24.1",
            "2021",
            "2153454.81",
            "0.00"
          ],
          "rowNumber": 1588,
          "rowPosition": 1589,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1589\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50300",
            "1736",
            "35",
            "1209",
            "17546",
            "",
            "459",
            "4174",
            "1510 06.181.005.4025.1 4 4 90 52 1.0.0",
            "2021",
            "0.00",
            "415749.75"
          ],
          "rowNumber": 1650,
          "rowPosition": 1651,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1651\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50300",
            "1736",
            "35",
            "1209",
            "17546",
            "",
            "459",
            "4174",
            "1510 06.181.005.4025.1 4 4 90 52 1.24.1",
            "2021",
            "19996677.30",
            "0.00"
          ],
          "rowNumber": 1651,
          "rowPosition": 1652,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1652\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50283",
            "1731",
            "57",
            "328",
            "17585",
            "",
            "498",
            "4159",
            "1250 06.181.034.4048.1 3 1 90 12 1.0.0",
            "2021",
            "0.00",
            "2000.00"
          ],
          "rowNumber": 1675,
          "rowPosition": 1676,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1676\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50283",
            "1731",
            "57",
            "328",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.70.1",
            "2021",
            "10000.00",
            "0.00"
          ],
          "rowNumber": 1676,
          "rowPosition": 1677,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1677\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50300",
            "1735",
            "35",
            "1209",
            "17546",
            "",
            "459",
            "4174",
            "1510 06.181.005.4025.1 4 4 90 52 1.0.0",
            "2021",
            "0.00",
            "467414.08"
          ],
          "rowNumber": 3495,
          "rowPosition": 3496,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"3496\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50300",
            "1735",
            "35",
            "1209",
            "17546",
            "",
            "459",
            "4174",
            "1510 06.181.005.4025.1 4 4 90 52 1.24.1",
            "2021",
            "21998833.92",
            "0.00"
          ],
          "rowNumber": 3516,
          "rowPosition": 3517,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"3517\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50295",
            "1733",
            "57",
            "857",
            "17585",
            "",
            "469",
            "4159",
            "1250 06.181.034.4048.1 3 3 90 30 1.70.1",
            "2021",
            "3779932.81",
            "0.00"
          ],
          "rowNumber": 3611,
          "rowPosition": 3612,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"3612\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50295",
            "1733",
            "57",
            "857",
            "17585",
            "",
            "534",
            "4159",
            "1250 06.181.034.4048.1 3 3 90 39 1.70.1",
            "2021",
            "1668973.02",
            "0.00"
          ],
          "rowNumber": 3612,
          "rowPosition": 3613,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"3613\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50295",
            "1733",
            "57",
            "857",
            "17585",
            "",
            "459",
            "4159",
            "1250 06.181.034.4048.1 4 4 90 52 1.70.1",
            "2021",
            "2526324.52",
            "0.00"
          ],
          "rowNumber": 3613,
          "rowPosition": 3614,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"3614\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        }
      ],
      "stats": {
        "errors": 34
      },
      "valid": false
    }
  ],
  "stats": {
    "errors": 34,
    "tasks": 1
  },
  "valid": false
}